shared/common-adapters/zoomable-box.ios.tsx

```diff
@@ -127,6 +127,11 @@ export const ZoomableBox = (props: Props) => {
     })
   const taps = Gesture.Exclusive(doubleTap, singleTap)
 
+  // this is nasty but i need a way to force this component to render correctly. Passing new zoomScale
```

Comment:
there are 2 underyling issues i think with the native component. If the src changes you can get a really weird rendering artifact where it seems like its zoomed in a tremendous amount and you can't zoom out. Fixed this by entirely using a new key on the scale change. (below). On this line we need to 'jiggle' the zoomscale so it'll adopt it. If we don't do this you can log and see the scale is set and passed down but its not respected (the onScroll callback fires but not with this value)